gems/actionview/6.0.3.2/actionview-generated.rbs

```diff
@@ -9110,6 +9110,8 @@ module ActionView
 
     def find_all_with_query: (untyped query) -> (untyped | ::Array[untyped])
 
+    def each: () { (untyped) -> void } -> untyped
```

Comment:
`#each` was missing because it is delegated.

https://github.com/rails/rails/blob/6-0-stable/actionview/lib/action_view/path_set.rb#L16